src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -5,6 +5,9 @@ based on: https://www.codedesigntips.com/2021/06/28/swagger-ui-with-login-form-a
     window.addEventListener('load', () => setTimeout(initLoginForm, 0), false);
 })();
 
+const AUTHORIZATION_TOKEN_KEY = 'access_token';
```

Comment:
since you've changed the mechanism to use cookies, it's better to rename this constant to `ACCESS_TOKEN_COOKIE_NAME`